llvm/lib/Support/BLAKE3/CMakeLists.txt

```diff
@@ -81,3 +81,8 @@ endif()
 add_library(LLVMSupportBlake3 OBJECT EXCLUDE_FROM_ALL ${LLVM_BLAKE3_FILES})
 set_target_properties(LLVMSupportBlake3 PROPERTIES FOLDER "LLVM/Libraries")
 llvm_update_compile_flags(LLVMSupportBlake3)
+
+# We have to manually set this because this library was not created with llvm_add_library
```

Comment:
Do we know why it doesn't use llvm_add_library?

Reply:
My best guess is because it a bunch of C code and assembly files that need custom file options